wurst/systems/meatsystem/MeatSystem.wurst

```diff
@@ -320,16 +325,16 @@ function onTargetOrder()
         and u.isInventoryFull()
         and (meatInventoryInstances.has(u) or isUnitMeatCarrier(u)))
 
-        MeatInventory instance = null
+        MeatInventory _instance = null
```

Comment:
Is `instance` a reserved word now? I'm not clear on why you add to add the underscore to the name.